django_rest_passwordreset/views.py

```diff
@@ -185,7 +202,22 @@ def post(self, request, *args, **kwargs):
                 # let whoever receives this signal handle sending the email for the password reset
                 reset_password_token_created.send(sender=self.__class__, instance=self, reset_password_token=token)
         # done
-        return Response({'status': 'OK'})
+        message = get_response_message("PASSWORD_REQUEST_ACCEPT")
+        return Response({"status_code": 200, "status": "OK", "message": message})
+
+
+def get_response_message(message_key):
+    """
+        A function return status message based on conditions
+        conditions: if message attribute define in settings then it will 
+                    return user define message otherwise, return static defined message
+        params: message_key
+        return: status_message
```

Comment:
Please change to:
```Returns configurable status messages
If message_key exists in settings.py, the corresponding message will be returned. Otherwise message_key
is returned
:param message_key: Key of the message in settings.py
:return: status_message
```